### PR TITLE
Package rename

### PR DESCRIPTION
**Pull request by qubixes**

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -25,8 +25,8 @@ identifiers:
   - type: doi
     value: 10.5281/zenodo.7696031
     description: Latest archived version of Metasynth
-repository-code: 'https://github.com/sodascience/metasynth'
-url: 'https://metasynth.readthedocs.io/'
+repository-code: 'https://github.com/sodascience/metasyn'
+url: 'https://metasyn.readthedocs.io/'
```
Comment (Samuwhale) [inline CITATION.cff:29]: This link is currently broken - do we have to manually make a rtd page for metasyn? or will the current page automatically be renamed?

File: docs/source/usage/extensions.rst [hunk 2]
```diff
@@ -1,36 +1,36 @@
...
+Metasyn is developed with a strong focus on extensibility, allowing developers to easily build upon its existing functionality. This page aims to document major extensions that are currently available.
 
 .. note:: 
-    Refer to our :doc:`/developer/developer` for more information on how to develop (additional functionality) for MetaSynth.
+    Refer to our :doc:`/developer/developer` for more information on how to develop (additional functionality) for metasyn.
 
 Disclosure control
 ------------------
-`Disclosure Control <https://github.com/sodascience/metasynth-disclosure-control>`_ is a plugin developed in-house for MetaSynth.
+`Disclosure Control <https://github.com/sodascience/metasyn-disclosure-control>`_ is a plugin developed in-house for metasyn.
```
Comment (Samuwhale) [inline docs/source/usage/extensions.rst:10]: This link is currently also broken, we should not forget to rename the disclosure control package after merging this PR.

**Review by Samuwhale — COMMENTED** [review]
Looks good to me! We have to be careful that URLs on other pages get updated too (disclosure control, rtd). But other than that, seems good. Nice work :)

File: CITATION.cff (hunk 1 above)
Comment (qubixes) [reply CITATION.cff:29]: Yes, I think @vankesteren needs to do that manually.

File: docs/source/usage/extensions.rst (hunk 2 above)
Comment (qubixes) [reply docs/source/usage/extensions.rst:10]: Thanks, I already have a rename PR for that package as well, but I haven't renamed the repo yet.

**Pull request merged**